pretix_sepadebit/payment.py

```diff
@@ -252,13 +253,31 @@ def shred_payment_info(self, obj: Union[OrderPayment, OrderRefund]):
         obj.info_data = d
         obj.save(update_fields=['info'])
 
+    @staticmethod
+    def norm(s):
+        return s.strip().upper().replace(" ", "")
+
     def execute_refund(self, refund: OrderRefund):
         if refund.payment.sepaexportorder_set.exists():
-            raise PaymentException('Already exported.')
-        refund.done()
+            refund.info_data = {
+                'payer': refund.payment.info_data['account'],  # Use payer to keep it compatible with the banktransfer-refunds
+                'iban': self.norm(refund.payment.info_data['iban']),
+                'bic': self.norm(refund.payment.info_data['bic']),
+            }
+            refund.save(update_fields=["info"])
+        else:
+            refund.done()
 
     def payment_refund_supported(self, payment: OrderPayment) -> bool:
-        return not payment.sepaexportorder_set.exists()
```

Comment:
Well, this can basically stay, right? If it's not been exported yet, export is always possible regardless of valid IBAN. This is especially important to get out of invalid IBAN payments.

Reply:
Please have a look at the latest commit to see if I understood your change request correctly.

Reply:
Almost. Shouldn't it be ``return True``? If the debit never made it to the bank, a "refund" is always possible.